### PR TITLE
fix: redis timeout while running integration tests

### PR DESCRIPTION
**Pull request by kirgene**
fix redis timeout while running integration tests

### CODE REVIEW

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts [hunk 1]
```diff
@@ -73,16 +73,19 @@ const bulkTransactionEntityRepo = new RedisBulkTransactionStateRepo(bulkTransact
...
 
 describe("Tests for Outbound Command Event Handler", () => {
-
   beforeEach(async () => {
+    domainEvents = [];
+  });
+
+  beforeAll(async () => {
     await producer.init();
     domainEvents = [];
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:82]:
```suggestion
```

Already handled by the `beforeEach`.

**Review by mdebarros — CHANGES_REQUESTED** [review]
+1

File: .circleci/config.yml [hunk 2]
```diff
@@ -205,7 +205,17 @@ jobs:
       - run:
           name: Execute Integration Tests
           command: |
-            yarn run test:integration
+            run_int_tests() {
```
Comment (mdebarros) [inline .circleci/config.yml:208]: Why are we not using docker-compose to start up the handlers similar to how we do so with the api-svc?

I believe everything should work as expected if we used the existing mechanism.

**Review by mdebarros — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (kirgene) [reply .circleci/config.yml:208]: running all services altogether breaks integration tests.
services start to communicate with each other, and test consumers get only a portion of all messages

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**